### PR TITLE
Proposed small fix to get the team identifier working in iOS 9.2

### PR DESCRIPTION
**Pull request by danwalkowski**
We need to have a discussion about this; it seems that Apple broke access to poking around in the kSecClass:  classes, (iOS 92. latest Xcode) which we were using to get the Team ID.

I found another way, but it requires the partner to add a key to their plist.

### CODE REVIEW

**Comment by danwalkowski** [comment]
additional info: the plist key is filled in at build time

File: Branch-TestBed/Branch-TestBed/Branch-TestBed-Info.plist [hunk 1]
```diff
@@ -45,6 +45,8 @@
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
 	</array>
+	<key>AppIdentifierPrefix</key>
+	<string>$(AppIdentifierPrefix</string>
```
Comment (derrickstaten) [inline Branch-TestBed/Branch-TestBed/Branch-TestBed-Info.plist]: should this be `$(AppIdentifierPrefix)`? seems weird to have just an open parens

Comment (derrickstaten) [inline Branch-TestBed/Branch-TestBed/Branch-TestBed-Info.plist]: also, is this the required plist entry?

**Comment by derrickstaten** [comment]
:+1:

**Pull request merged**